CLAUDE.md

```
@@ -0,0 +1,60 @@
+### 프로파일
+
+| 프로파일 | 환경변수 파일 | DB ddl-auto |
+|---|---|---|
+| `local` | `.env.local` | update |
+| `production` | `.env.production` | validate |
+
+`DotEnvConfig`가 기동 시 `.env.{profile}` 로드 → 시스템 프로퍼티 등록 (시스템 환경변수 우선).
+
+### 주요 환경변수
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)